src/bootstrap/config.rs

```diff
@@ -200,6 +200,12 @@ pub struct Config {
     pub initial_rustc: PathBuf,
     pub initial_rustfmt: Option<PathBuf>,
     pub out: PathBuf,
+
+    // Clippy lint levels
+    pub clippy_lint_allow: Vec<String>,
+    pub clippy_lint_deny: Vec<String>,
+    pub clippy_lint_warn: Vec<String>,
+    pub clippy_lint_forbid: Vec<String>,
```

Comment:
These are only used for clippy. Can you move them to `Subcommand::Clippy` so people don't use them in other subcommands by accident? That also allows removing them from `Config` and `Flags`.

Reply:
Done. Not sure what `r=` means, sorry